src/components/Settings/AllSettingsFields.tsx

```diff
@@ -13,7 +13,7 @@ import { isDesktop } from 'lib/isDesktop'
 import { ActionButton } from 'components/ActionButton'
 import { SettingsFieldInput } from './SettingsFieldInput'
 import toast from 'react-hot-toast'
-import { APP_VERSION } from 'routes/Settings'
+import { APP_VERSION, APP_NAME } from 'routes/Settings'
```

Comment:
Future tweak we should get these constant-like things out of routes/Settings that was probably some hairbrained thing I did while building the settings out